client/components/web-preview/content.jsx

```diff
@@ -255,7 +261,9 @@ export default class WebPreviewContent extends Component {
 	}
 
 	render() {
-		const { translate, toolbarComponent: ToolbarComponent } = this.props;
+		const { translate, toolbarComponent: ToolbarComponent, fetchPriority, autoHeight } = this.props;
+		const isLoaded =
+			this.state.loaded && ( ! autoHeight || ( autoHeight && this.state.viewport !== null ) );
```

Comment:
We don't need to check `autoHeight` twice.

```suggestion
			this.state.loaded && ( ! autoHeight || this.state.viewport !== null );
```

Reply:
I thought it would make to code clearer, but I can change it for sure 😅